lib/vmdb/configuration_encoder.rb

```diff
@@ -22,7 +22,7 @@ def self.symbolize!(h)
     def self.load(data, symbolize_keys = true, &block)
       return {} if data.blank?
 
-      if defined?(Rails) && !Rails.env.production?
+      if !defined?(Rails) || !Rails.env.production? || ENV['ERB_IN_CONFIG']
```

Comment:
it will go inside `if` statement if `defined?(Rails)` evaluates to truthy, but did not before.  Is that the intent?